doc/docs/Mode_Decomposition.md

```diff
@@ -180,61 +176,64 @@ These functions are implemented in [src/dft.cpp](https://github.com/NanoComp/mee
 
 ## How Mode Decomposition Works
 
-The theoretical basis of the mode-decomposition algorithm is the orthogonality relation satisfied by the normal modes:
+The theoretical basis of the mode-decomposition algorithm is an orthogonality relation satisfied by the eigenmodes:
 
-$$ \left\langle \mathbf{E}_m^{\sigma} \right|
-   \left.       \mathbf{H}^\tau_n     \right\rangle
-   =C_{m}\delta_{mn}\delta_{\sigma\tau} 
+$$
+\left\langle \psi_m^\sigma, \psi_m^\tau_n \right\rangle
+=C_{m}\delta_{mn}\delta_{\sigma\tau}
    \qquad \{\sigma,\tau\}\in\{+,-\}
 $$
 
-where the inner product involves an integration over transverse coordinates:
+where the (indefinite) inner product involves an integration over transverse coordinates:
 
-$$ \left\langle \mathbf{f} \right| \left. \mathbf{g} \right\rangle 
+$$ \left\langle \psi , \psi' \right\rangle
    \equiv
    \int_{S}
-    \Big[ \mathbf{f}^*(\vec \rho) \times \mathbf{g}(\vec \rho)\Big]
+    \Big[ \mathbf{E}^*(\vec \rho) \times \mathbf{H}'(\vec \rho) + \mathbf{E}'(\vec \rho) \times \mathbf{H}^*(\vec \rho)\Big]
```

Comment:
For clarity, perhaps the `'` superscript for the `E` and `H` fields should be explicitly defined in the text?